Dockerfile

```diff
@@ -1,24 +1,29 @@
-FROM golang:1.22-bookworm as builder
+FROM golang:1.22-bullseye as builder
 
 WORKDIR /app
 ADD . /app
 
-RUN apt-get update \
-    && DEBIAN_FRONTEND=noninteractive \
-    apt-get install --no-install-recommends --assume-yes \
-    wget git make curl libnl-3-dev libnet-dev libbsd-dev runc libcap-dev \
-    libgpgme-dev btrfs-progs libbtrfs-dev libseccomp-dev libapparmor-dev libprotobuf-dev \
-    libprotobuf-c-dev protobuf-c-compiler protobuf-compiler python3-protobuf software-properties-common zip
+RUN <<EOT
+set -eux
+DEBIAN_FRONTEND=noninteractive
+apt-get update
+apt-get install -y wget git make curl libnl-3-dev libnet-dev libbsd-dev runc libcap-dev
```

Comment:
Why do this instead of how we were installing before?

Reply:
Just cleaner, plus allows us to `set -eux`, so on failure during build, we get better message.

Reply:
Thought it's quite tedious to keep using `&&` you know.